client/src/components/HeroSlider/HeroSlider.jsx

```diff
@@ -23,7 +23,7 @@ export default function HeroSlider() {
     } catch (err) {
       console.error(err);
     }
-  });
+  }, [express]);
```

Comment:
you can move the `express` variable inside of `useEffect` to avoid having to add it as a dependency